File: Lab3/linearSearch.java
import java.util.Scanner;

public class linearSearch {
    public static void main(String[] args) {
        int arr[] = {6,2,3,1,5,1};
        Scanner scanner = new Scanner(System.in);
        System.out.println("enter number : ");
        int element = scanner.nextInt();
        for(int i=0;i<arr.length;i++){
            if(arr[i] == element){
                System.out.println("index = "+i);
            }
        }
        scanner.close();
    }
    
}

File: DS/lab1/linearSearch.c
#include<stdio.h>

void main(){

    int length = 0;
    int arr[50];
    int element;
    printf("enter the length of the array : ");
    scanf("%d",&length);
    printf("the element to be found : ");
    scanf("%d",&element);

    for(int i=0;i<length;i++){
        printf("enter element %d : ",i);
        scanf("%d",&arr[i]);
    }
    for(int j = 0;j<length;j++){
        if(arr[j] == element){
            printf("index is : %d\n",j);
        }
    }
}
